ruby/ql/lib/change-notes/2022-03-09-http-client-getaurlpart.md

```diff
@@ -0,0 +1,4 @@
+---
+category: breaking
+---
+* The `getURL` member-predicate of the `HTTP::Client::Request` and `HTTP::Client::Request::Range` classes from `Concepts.qll` have been renamed to `getAUrlPart`.
```

Comment:
We need either one of the following, to make the subject and verb agree:
- "The `getURL` member-predicates … have been …"
- "The `getURL` member-predicate … has been …"

Reply:
:ok_hand: 

```suggestion
* The `getURL` member-predicates of the `HTTP::Client::Request` and `HTTP::Client::Request::Range` classes from `Concepts.qll` have been renamed to `getAUrlPart`.
```